C/casacore/build_tarballs.jl

```diff
@@ -0,0 +1,90 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "casacore"
+version = v"3.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/casacore/casacore.git", "a735bf5f31ea012ac4f7f7378a1f89c1fc136a06")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+OPENBLAS=(-lopenblas64_)
+
+cd $WORKSPACE/srcdir/casacore
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DBUILD_PYTHON=OFF \
+      -DBUILD_SHARED_LIBS=ON \
+      -DUSE_READLINE=OFF \
+      -DUSE_HDF5=ON \
+      -DFFTW3_DISABLE_THREADS=ON \
+      -DBLAS_LIBRARIES="${OPENBLAS[*]}" \
+      ..
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc")
+]
```

Comment:
It'd be nice to build for all platforms, not just yours :slightly_smiling_face: Also, note the warning:
```
┌ Warning: /agent/_work/1/s/C/casacore/build/x86_64-linux-gnu/b6utXa8y/x86_64-linux-gnu-libgfortran5-cxx11/destdir/bin/findmeastable contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/EHPCr/src/auditor/compiler_abi.jl:242
```
Finally, since we're requiring julia v1.6 (see last line of the file), we can build for other platforms, including `aarch64-apple-darwin`:
```suggestion
platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
```